apps/wallet-mobile/src/features/Exchange/common/useProviders.tsx

```diff
@@ -0,0 +1,22 @@
+import {Providers} from '@yoroi/exchange'
+import {Exchange} from '@yoroi/types'
+import * as React from 'react'
+
+import {useExchange} from './ExchangeProvider'
+
+export const useProviders = () => {
```

Comment:
```suggestion
export const useExchangeProvidersByFeatures = (predicate?: 'buy' 
| 'sell') => {
```

Reply:
I changed completely this hook. Please have a look